packages/editor/src/store/actions.native.js

```diff
@@ -25,3 +30,12 @@ export function togglePostTitleSelection( isSelected = true ) {
 export function* autosave() {
 	RNReactNativeGutenbergBridge.editorDidAutosave();
 }
+
+/**
+ * Action generator to set the selected layout template.
+ *
+ * @param {Object} template Layout's template
+ */
+export function* setLayoutTemplate( template ) {
```

Comment:
@geriux I originally created my branch from yours but merged master after yours was merged 😅 Is this expected to be here or is it a bad merge?

Reply:
Oh no, this isn't supposed to be here =(

Reply:
👍 Reverted those changes in 8af053d8ff